get_all_manifests.sh

```diff
@@ -30,6 +29,9 @@ wget -q -c ${MANIFESTS_TARBALL_URL} -O - | tar -zxv -C ./.odh-manifests-tmp/ --s
 cp -r ./.odh-manifests-tmp/model-mesh/ ./odh-manifests
 cp -r ./.odh-manifests-tmp/odh-model-controller/ ./odh-manifests
 cp -r ./.odh-manifests-tmp/modelmesh-monitoring/ ./odh-manifests
+cp -r ./.odh-manifests-tmp/odh-notebook-controller/ ./odh-manifests
```

Comment:
do we need this for workbench? see line 13

Reply:
think you will need a rebase from "main" to get the change, like line 11 and 14 are duplicated.